### PR TITLE
doc: update references to point-of-sale

### PR DESCRIPTION
**Pull request by Shabirmean**
#### Description
- After bringing the ansible scripts for `anthos-baremetal-edge-deployment` into this repo we have to update references to old folder/repo 
- **Any changes that seem related to updating references are linting changes that were fixed and reviewed in https://github.com/GoogleCloudPlatform/anthos-samples/pull/293**

#### Change summary
- Update references to `anthos-baremetal-edge-deployment` -> `anthos-bm-edge-deployment`
- Update references to the point of sale repo where appropriate

#### Related PRs/Issues
- https://github.com/GoogleCloudPlatform/anthos-samples/pull/292
- https://github.com/GoogleCloudPlatform/anthos-samples/pull/293

### CODE REVIEW

File: anthos-bm-edge-deployment/README.md [hunk 1]
```diff
@@ -62,10 +62,10 @@ The following quick start guide will take approximately **55-60 minutes** to com
 
 ```sh
 # once you have forked this repository, clone it to your local machine
-git clone https://github.com/<YOUR_GITHUB_USERNAME>/point-of-sale
+git clone https://github.com/<YOUR_GITHUB_USERNAME>/anthos-samples
 
 # move into the root of the infrastructure setup directory
-cd point-of-sale/anthos-baremetal-edge-deployment
+cd anthos-samples/anthos-bm-edge-deployment
```
Comment (bourgeoisor) [inline anthos-bm-edge-deployment/README.md:68]: What's the context behind changing the name? Is "anthos bm" well understood by the community, or would it be better to keep the longform?

Comment (Shabirmean) [reply anthos-bm-edge-deployment/README.md:68]: The reduction from `baremetal` to `bm` is to make it consistent with the rest of the directories in this repo
- `anthos-bm-gcp-terraform`
- `anthos-bm-openstack-terraform`

Comment (bourgeoisor) [reply anthos-bm-edge-deployment/README.md:68]: Gotcha, thank you.

**Review by bourgeoisor — APPROVED** [review]
LGTM

**Pull request merged**